docs/tenants/zz_metadata_schema_generated.md

```diff
@@ -135,7 +135,7 @@
 
 - **`indexImage`** *(string)*
 
-- **`addonImageSetVersion`** *(string)*: A string which specifies the imageset to use. Can either be 'latest' or a version string).
+- **`addonImageSetVersion`**: A string which specifies the imageset to use. Can either be 'latest' or a version string.
```

Comment:
```suggestion
- **`addonImageSetVersion`** *(string)*: Specifies the imageset to use. Can either be 'latest' or a version string.
```

Reply:
The might be a little more consistent with the structure used for the other fields

Reply:
thanks for the review. :) I get your point, but this corresponds to the change that Amador did in #196, where `type: string` was removed from the `metadata.schema.yaml` (https://github.com/mt-sre/managed-tenants-cli/pull/196/commits/73ef3689450f453f86e84e5ed0b6cf4692c4cd73#diff-ad3eac1137a13a51306b5b5493dc7723592bf3415b304eae3b5eaece2733fb81L267). 

This is an auto-generated change, if you run `make generate`, you'll also get this change. I got this in a different PR, and this change did not align with the PR's context. So I decided to create another PR for this.

Reply:
Could you maybe also change it in `managedtenants/schemas/metadata.schema.yaml`? :)

Reply:
done!